test/conformance/memory-migrate/CMakeLists.txt

```diff
@@ -0,0 +1,4 @@
+if (UR_DPCXX)
```

Comment:
With the move of `add_subdirectory(memory-migrate)` this check will no longer be necessary.

Reply:
Thanks change made